binders/kafka-binder/spring-cloud-stream-binder-kafka-reactive/src/test/resources/logback.xml

```diff
@@ -1,20 +1,20 @@
 <configuration>
-    <appender name="stdout" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder>
-            <pattern>%d{ISO8601} %5p %t %c{2}:%L - %m%n</pattern>
-        </encoder>
-    </appender>
-    <logger name="org.apache.kafka" level="WARN"/>
-    <logger name="reactor.kafka" level="DEBUG"/>
-    <logger name="org.springframework.integration.kafka" level="INFO"/>
-    <logger name="org.springframework.kafka" level="DEBUG"/>
-    <logger name="org.springframework.cloud.stream" level="INFO" />
-    <logger name="org.springframework.integration.channel" level="DEBUG" />
+	<appender name="stdout" class="ch.qos.logback.core.ConsoleAppender">
+		<encoder>
+			<pattern>%d{ISO8601} %5p %t %c{2}:%L - %m%n</pattern>
+		</encoder>
+	</appender>
+	<logger name="org.apache.kafka" level="WARN"/>
+	<logger name="reactor.kafka" level="DEBUG"/>
+	<logger name="org.springframework.integration.kafka" level="INFO"/>
+	<logger name="org.springframework.kafka" level="DEBUG"/>
+	<logger name="org.springframework.cloud.stream" level="INFO" />
+	<logger name="org.springframework.integration.channel" level="DEBUG" />
 	<logger name="kafka.server.ReplicaFetcherThread" level="ERROR"/>
 	<logger name="kafka.server.LogDirFailureChannel" level="FATAL"/>
 	<logger name="kafka.server.BrokerMetadataCheckpoint" level="ERROR"/>
 	<logger name="kafka.utils.CoreUtils$" level="ERROR"/>
-    <root level="WARN">
-        <appender-ref ref="stdout"/>
-    </root>
+	<root level="WARN">
+		<appender-ref ref="stdout"/>
+	</root>
```

Comment:
I think we can defer this file refactoring to some other commit: the PR is already too complex to have this extra noise as well.